core/src/main/resources/error/error-classes.json

```diff
@@ -3783,6 +3783,12 @@
       "Failed to execute command because subquery expressions are not allowed in DEFAULT values."
     ]
   },
+  "_LEGACY_ERROR_TEMP_1348" : {
```

Comment:
Could you assign proper name to the error class, please. The `_LEGACY_ERROR_TEMP_` prefix was used for migration purpose in batch way to switch to error classes in all existing exceptions.